src/front/client/index.html

```diff
@@ -533,155 +645,143 @@ <h2 class="starter-modal__title">
           </span>
           <button class="btn btn-primary btc-open-in-new-tab">
-            <a
-              href="https://wallet.wpmix.net"
-              id="onFailLocalStorageLink"
-              target="_blank"
-            >
-              Open App in new tab
-            </a>
+            <a href="https://wallet.wpmix.net" id="onFailLocalStorageLink" target="_blank"> Open App in new tab </a>
           </button>
         </div>
         <div id="usersInform" class="usersInform"></div>
       </div>
       <script>
-        var localStorageIsOk = true;
+        let localStorageIsOk = true
+
         try {
-          var testLocalStorage = window.localStorage;
+          const testLocalStorage = window.localStorage
         } catch (e) {
-          localStorageIsOk = false;
-          document.getElementById('onFailLocalStorageLink').href =
-            window.location.href;
-          document
-            .getElementById('onFailLocalStorageMessage')
-            .classList.remove('d-none');
-          var sendErrorFeedback = function () {
-            var msg = 'localStorage error: agent(' + navigator.userAgent + ')';
-            msg += ', location(' + window.location.href + ')';
-
-            var url =
-              'https://noxon.wpmix.net/counter.php?msg=' + encodeURI(msg);
+          localStorageIsOk = false
+          document.getElementById('onFailLocalStorageLink').href = window.location.href
+          document.getElementById('onFailLocalStorageMessage').classList.remove('d-none')
+
+          const sendErrorFeedback = function () {
+            let msg = 'localStorage error: agent(' + navigator.userAgent + ')'
+            msg += ', location(' + window.location.href + ')'
+
+            let url = 'https://noxon.wpmix.net/counter.php?msg=' + encodeURI(msg)
 
             window.jQuery.ajax({
               type: 'POST',
               url: url,
-            });
-          };
+            })
+          }
+
           if (!window.jQuery) {
             // inject jQuery for send request to counter for feedback
-            var jsScriptTag = document.createElement('SCRIPT');
-            jsScriptTag.src = 'https://code.jquery.com/jquery-3.5.1.min.js';
-            console.log('jsScriptTag', jsScriptTag);
-            document.getElementsByTagName('BODY')[0].appendChild(jsScriptTag);
-            var waitJQLoad = function (onLoaded) {
+            let jsScriptTag = document.createElement('SCRIPT')
+            jsScriptTag.src = 'https://code.jquery.com/jquery-3.5.1.min.js'
+            console.log('jsScriptTag', jsScriptTag)
+            document.getElementsByTagName('BODY')[0].appendChild(jsScriptTag)
+
+            let waitJQLoad = (onLoaded) => {
               if (window.jQuery) {
-                onLoaded();
+                onLoaded()
               } else {
-                window.setTimeout(function () {
-                  waitJQLoad(onLoaded);
-                }, 100);
+                window.setTimeout(() => {
```

Comment:
there no ecma - its native script with back-compatibility. revert code and return back dot with comma